### ISSUE TITLE
Pipe server 500 errors somewhere sane, show pretty error page

### ISSUE THREAD

**Issue by gstuder-ona** [onaio/superset-patchup#36]
Probably pipe to a sentry instance?  May need additional config too.

Any ideas for what a pretty error page looks like?

**Comment by moshthepitt** [comment]
> Probably pipe to a sentry instance? May need additional config too.

Yeah, I think sending the errors to sentry will require extra configs, there is an issue for that in the ansible role: https://github.com/onaio/ansible-superset/issues/32

> Any ideas for what a pretty error page looks like?

Idk, I guess anything that works with minimal css/js. [Some](https://github.com/django/django/blob/42b9a23267f14be39b9b00958e18d5746783208e/django/contrib/admin/templates/admin/500.html) [ideas](https://uicookies.com/500-error-page-templates/).
